doc/en/how-to/cache.rst

```diff
@@ -176,14 +176,14 @@ with more recent files coming first.
 Behavior when no tests failed in the last run
 ---------------------------------------------
 
-When no tests failed in the last run, or when no cached ``lastfailed`` data was
-found, ``pytest`` can be configured either to run all of the tests or no tests,
-using the ``--last-failed-no-failures`` option, which takes one of the following values:
+This option governs the behavior of ``--last-failed``.
+Determines whether to execute tests when there are no previously (known)
+failures or when no cached ``lastfailed`` data was found.
 
 .. code-block:: bash
 
-    pytest --last-failed --last-failed-no-failures all    # run all tests (default behavior)
-    pytest --last-failed --last-failed-no-failures none   # run no tests and exit
+    pytest --last-failed --last-failed-no-failures all    # run the full test suite again (default behavior)
```

Comment:
I don't think we need to use a command line example, perhaps just listing the options would be better, something like:

```rst
* ``all``:  when there are no known test failures, runs all tests (the full test suite again). This is the default.
* ``none``: when there are no known test failures, just emits a message stating this and exit successfully.
```

What do you think?

Reply:
I don't think it would be more confusing to include the command line example, and it hones in the idea of how to use the command and that it is utilized with the option `--last-failed` 

I think that the current documentation is a bit more concise in that the above docstring details what the command does and that the two command line examples show the two options with the comment giving enough information that a user should understand what they both do. 

However, I do think that  I can add a bit more to the current version of it, which I will do soon.

Reply:
You got a point, but that leads to hard-to-read horizontal bar:

![image](https://github.com/pytest-dev/pytest/assets/1085180/4c7f6aa6-63a3-4db1-84ce-474bd7b70939)

https://pytest--11361.org.readthedocs.build/en/11361/how-to/cache.html

That's why I think perhaps listing the options explicitly would be better... perhaps we can have both? In the command-line example, use a shorter explanation (the previous one is good I think), then we can use the long form in the listing.

Reply:
That sounds good to me. 👍

Reply:
Looks great!

![image](https://github.com/pytest-dev/pytest/assets/1085180/4a6a9d6a-5843-477e-aefd-ac98d3b50a5d)

https://pytest--11361.org.readthedocs.build/en/11361/how-to/cache.html